frontend/__tests__/unit/pages/Projects.test.tsx

```
@@ -22,13 +27,29 @@ jest.mock('components/Pagination', () =>
   ))
 )
+jest.mock('wrappers/FontAwesomeIconWrapper', () => ({
+  __esModule: true,
+  default: () => <span data-testid="mock-icon" />,
+}))
+
+jest.mock('@/components/MarkdownWrapper', () => {
+  return ({ content, className }: { content: string; className?: string }) => (
+    <div
+      className={`md-wrapper ${className || ''}`}
+      dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)